microsoft-edge/webview2/concepts/webview-features-flags.md

```diff
@@ -58,6 +58,8 @@ See also Globals:
 
 The following are some of the flags we've seen used.
 
+If the flag is enabled, such as by setting it to `true`, the following behavior results.
```

Comment:
```suggestion
If the flag is enabled, the following behavior results.
```

Are flags enabled by setting them to `true`? It's likely that not all flags take a boolean value, so I'm wondering if omitting this part would be better.